### PR TITLE
Provide `t_min` and `t_max` for flux injection

### PR DESCRIPTION
**Pull request by RemiLehe**
In some simulations, we might want to interrupt particle flux injection after some time. 
This PR implements the corresponding option.

### CODE REVIEW

**Comment by dpgrote** [comment]
Since you are adding `tmax`, perhaps also add `tmin`.

File: Docs/source/usage/parameters.rst [hunk 1]
```diff
@@ -612,7 +612,8 @@ Particle initialization
       ``<species_name>.surface_flux_pos`` (`double`, location of the injection plane [meter])
       ``<species_name>.flux_normal_axis`` (`x`, `y`, or `z` for 3D, `x` or `z` for 2D, or `r` or `z` for RZ)
       ``<species_name>.flux_direction`` (`-1` or `+1`, direction of flux relative to the plane)
-      ``<species_name>.num_particles_per_cell`` (`double`)
+      ``<species_name>.num_particles_per_cell`` (`double`). In addition, the injection of particles
```
Comment (dpgrote) [inline Docs/source/usage/parameters.rst]: To be consistent in the documentation, perhaps do it this way:
```
      ``<species_name>.num_particles_per_cell`` (`double`)
      ``<species_name>.flux_tmax`` (`double`, Optional time at which the flux will be turned off. Ignored when negative.)
```

**Comment by dpgrote** [comment]
This looks good. I have the one comment as a suggestion for the documentation format.

Will you add the `tmin`?

**Comment by RemiLehe** [comment]
Thanks for your comments @dpgrote. I added `tmin` and updated the documentation accordingly.

**Review by dpgrote — APPROVED** [review]
Looks good! Thanks for making the fixes

**Pull request merged**